augur.config.json

```
@@ -0,0 +1,47 @@
+
+{
+    "Cache": {
+        "config": {
+            "cache.data_dir": "runtime/cache/",
+            "cache.lock_dir": "runtime/cache/",
+            "cache.type": "file"
+        }
+    },
+    "Database": {
+        "host": "hostname.io",
+        "name": "name",
+        "pass": "password",
+        "port": "0000",
+        "user": "username"
+     },
+    "Development": {
+        "developer": "0",
+        "interactive": "0"
+    },
+    "Facade": {
+        "host": "hostname.io",
+        "name": "name",
+        "pass": "password",
+        "port": "0000",
+        "projects": [
+            "project",
+            "project2",
+        ],
+        "user": "username"
+    },
+    "GitHub": {
+        "apikey": "0000000000000000000000000000000000"
+    },
+    "LibrariesIO": {
+        "apikey": ""
```

The connection to LibrariesIO is now through a database, so I think it would be fine to have this section mirror the earlier `Database` section, something like: 
```suggestion
                "host": "hostname.io",
         "name": "name",
         "pass": "password",
         "port": "0000",
         "user": "username"
```